tests/forcetorque/ForceTorqueTest.cc

```diff
@@ -0,0 +1,49 @@
+#include <gtest/gtest.h>
+#include <yarp/dev/PolyDriver.h>
+#include <yarp/dev/MultipleAnalogSensorsInterfaces.h>
+#include <yarp/os/Network.h>
+#include <gz/sim/TestFixture.hh>
+
+TEST(ForceTorqueTest, PluginTest)
+{
+  yarp::os::NetworkBase::setLocalMode(true);
+  
+  // Maximum verbosity helps with debugging
+  gz::common::Console::SetVerbosity(4);
+
+  // Instantiate test fixture
+  gz::sim::TestFixture fixture("../../../tests/forcetorque/model.sdf");
+
+  int iterations = 5000;
+  fixture.Server()->Run(/*_blocking=*/true, iterations, /*_paused=*/false);
+
+  yarp::os::Property option;
+  option.put("device","multipleanalogsensorsclient");
+  option.put("remote","/forcetorque");
+  option.put("timeout",5.0);
+  option.put("local", "/ForceTorqueTest");
+  yarp::dev::PolyDriver driver;
+
+  ASSERT_TRUE(driver.open(option));
+
+  yarp::dev::ISixAxisForceTorqueSensors* isixaxis = nullptr;
+
+  ASSERT_TRUE(driver.view(isixaxis));
+
+  fixture.Server()->Run(/*_blocking=*/true, iterations, /*_paused=*/false);
+  yarp::sig::Vector measure(6);
+  std::string sensorName;
+  double timestamp;
+  
+  isixaxis->getSixAxisForceTorqueSensorName(0, sensorName);
+  isixaxis->getSixAxisForceTorqueSensorMeasure(0, measure, timestamp);
```

Comment:
I suggest:
```suggestion
  size_t maxNrOfReadingAttempts = 20;
  bool readSuccessful = false;
  for (size_t i=0; (i < maxNrOfReadingAttempts) && !readSuccessful ; i++)
  {
      readSuccessful = isixaxis->getSixAxisForceTorqueSensorMeasure(0, measure, timestamp);
      if (readSuccessful)
  }
  ASSERT_TRUE(readSuccessful);
```

if the failure continue, you can inspect also the value returned by `getSixAxisForceTorqueSensorStatus` to understand why the output is not the expected one.